desktop/core/src/desktop/js/reactComponents/MetricsComponent/MetricsComponent.test.tsx

```diff
@@ -0,0 +1,117 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import { render, waitFor, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import MetricsComponent from './MetricsComponent';
+
+// Mock matchMedia for compatibility with certain UI libraries
```

Comment:
matchMedia error is coming to more than places as told by Nidhi. She has asked me to add these lines of code and it is fixing the matchMedia error that comes on running `npm run test-dev`
We can remove these lines of code (line number 23-35) once Nidhi merges these matchMedia code in Hue.